accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+1. If we get here for `disable`, then fail.
+1. Exclude any lower versions.
+1. If `minor` or `latestMinor`, exclude any that don't match major version. 
+1. If `feature` or `latestFeature`, exclude any that don't match major and minor version.
+1. If `patch` or `latestPatch`, exclude any that don't match major, minor and feature band version.
+1. If there is one left, select it.
+1. If any of the `latest` options are used, select the highest. 
+1. Otherwise, select the lowest major, and within that lowest minor, and within that lowest feature band. Then select the highest patch in that feature band.  
+
+## Customer problems addressed
+
+* User wants to install a preview but only use it with a subset of projects.
+  * User opts out in projects where preview should not be used by adding `ignorePreview: true` to those projects, or
+  * User adds `ignorePreview: true` in a `global.json` higher in their file system, and adds `ignorePreview: false` in a `global.json` in specific projects.
+* User wants to always use a specific SDK patch version, and receive an error if it is not on the machine.
+  * User adds `rollForward: disable` to a `global.json`
+* User wants to always run the latest patch of a particular SDK feature band.
+  * User adds `rollForward: latestPatch` and the lowest version (with patch) they want to a `global.json`
+* User includes a feature that requires an SDK that corresponds to a SDK feature band or higher, and wants to run the best available tools.
+  * User adds `rollForward: latestMajor` and the lowest version (with patch) they want to a `global.json`
+* User wants to know what version of the SDK is being used.
+  * Not addressed in this proposal: `global.json` will still be a subtle, hidden and easily forgotten marker
+* User rarely wants the risk of running the unsupported scenario of multiple SDK versions in a solution build.
+  * Not addressed in this proposal: There is nothing that encourages users to place `global.json` in a logical location.
+* Hosted CI, including Azure DevOps and Kudu, need to provide a small number of SDKs.
+  * It will now be possible for users to specify `global.json` files in a way that can work with Azure DevOps (roll-forward across at least feature bands)
+  * We will work with Azure DevOps to determine the recommended `global.json` for this scenario.
+
+## Examples of usage
+
+This section lists a series of scenarios and following the corresponding `global.json`. Following this is a table that shows the SDK that would be selected with specific combinations of SDKs present on the machine.
+
+### 1. Explicitly state today's behavior when a `global.json` specifies a version
+
+The following `global.json` defines a specific version number, and to use a higher version if it is available.
+
+```json
+{
+  "sdk": {
+    "version": "2.2.100",
+    "ignorePreview": false,
+    "rollForward": "patch"
+  }
+}
+```
+
+The common `global.json` for this case is likely to be:
```

Comment:
What does "common" mean?

Reply:
It might be easier to read these examples if they were reversed ... minimal syntax and then the fully-specified syntax to clarify. I find this approach a bit confusing, and it also isn't clear which is recommended for practical use.